pkg/awsutils/awsutils.go

```diff
@@ -18,42 +18,40 @@ import (
 	"encoding/json"
 	"fmt"
 	"math/rand"
+	"net"
 	"os"
 	"regexp"
-	"strconv"
 	"strings"
 	"sync"
 	"time"
 
-	"github.com/pkg/errors"
-
-	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
-	"github.com/prometheus/client_golang/prometheus"
-
-	"github.com/aws/amazon-vpc-cni-k8s/pkg/ec2metadata"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/ec2wrapper"
+	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/logger"
 	"github.com/aws/amazon-vpc-cni-k8s/pkg/utils/retry"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
 	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
 	metadataMACPath      = "network/interfaces/macs/"
-	metadataAZ           = "placement/availability-zone/"
+	metadataAZ           = "placement/availability-zone"
 	metadataLocalIP      = "local-ipv4"
 	metadataInstanceID   = "instance-id"
 	metadataInstanceType = "instance-type"
 	metadataMAC          = "mac"
-	metadataSGs          = "/security-group-ids/"
-	metadataSubnetID     = "/subnet-id/"
-	metadataVPCcidrs     = "/vpc-ipv4-cidr-blocks/"
-	metadataDeviceNum    = "/device-number/"
-	metadataInterface    = "/interface-id/"
+	metadataSGs          = "/security-group-ids"
+	metadataSubnetID     = "/subnet-id"
+	metadataVPCcidrs     = "/vpc-ipv4-cidr-blocks"
+	metadataDeviceNum    = "/device-number"
+	metadataInterface    = "/interface-id"
 	metadataSubnetCIDR   = "/subnet-ipv4-cidr-block"
 	metadataIPv4s        = "/local-ipv4s"
```

Comment:
Btw, seems like these ones are only used in the tests now, maybe we should move them out?

Reply:
Oops, yes meant to include that patch too.  Done.